inc/user.class.php

```diff
@@ -3404,173 +3402,243 @@ static function getSqlSearchResult ($count = true, $right = "all", $entity_restr
                $right = [$right];
             }
             $forcecentral = true;
-            $where        = [];
 
             foreach ($right as $r) {
                switch ($r) {
                   case  'own_ticket' :
-                     $where[]= " (`glpi_profilerights`.`name` = 'ticket'
-                                  AND (`glpi_profilerights`.`rights` & ".Ticket::OWN.") ".
-                                  getEntitiesRestrictRequest("AND", "glpi_profiles_users", '',
-                                                             $entity_restrict, 1).") ";
+                     $WHERE[] = [
+                        'OR' => [
```

Comment:
Are you sure applying the `OR` here will act as a `OR` on all loop elements ? Maybe I am wrong, but as you have only one item (`AND`) in your `OR`, it will not be used.